2D3D_UnityProject/Assets/Scripts/Puzzles/Zodiac/ZodiacDisk.cs

```diff
@@ -56,7 +56,7 @@ public class ZodiacDisk : ZodiacPuzzlePiece
     private void Start()
     {
         // Print error if no correct symbol assigned for this disk
-        if(correctSymbol == null) {
+        if(correctSymbols == null) {
```

Comment:
we should add an additional check at the ZodiacPuzzle level that ensures each ZodiacDisk has the same number of "correct symbols" (i.e. number of times to solve the puzzle)